gatsby-node.js

```diff
@@ -1,23 +1,98 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
 const path = require(`path`);
+const fetch = require('node-fetch');
+const xmlParser = require('fast-xml-parser');
```

Comment:
that looks like the right choice for that simple xml 👍
https://www.npmjs.com/package/fast-xml-parser